README.md

```diff
@@ -2305,6 +2305,12 @@ integration, err := pipelinesManager.GetIntegrationByName("integrationName")
 integrations, err := pipelinesManager.GetAllIntegrations()
 ```
 
+#### Get All Raw Integrations
```

Comment:
Let's add this section to the Table of Contents at the top of the README, right after `Get All Integrations`.